app/dao/letter_branding_dao.py

```diff
@@ -0,0 +1,18 @@
+from app.models import LetterBranding
+
+
+def get_letter_branding_or_platform_default(domain=None):
+    letter_branding = None
+    if domain:
+        letter_branding = LetterBranding.query.filter(
+            LetterBranding.domain == domain
+        ).first()
+    if not letter_branding:
+        letter_branding = LetterBranding.query.filter(
+            LetterBranding.platform_default == True  # noqa
```

Comment:
this feels a bit odd. If I understand correctly, we only have one platform default letter branding that is always the same, right? If this is the case, do we have to query the whole table to find it?

I mean I guess it's not an expensive query since the table is tiny, so maybe it's ok...

Reply:
It's either that or have something in config. This seems to make sense, and it is a small table.

Reply:
ok 👍